packages/fluent-theme/src/fluent/styles/SearchBox.styles.ts

```diff
@@ -51,13 +51,13 @@ export const SearchBoxStyles = (props: ISearchBoxStyleProps): Partial<ISearchBox
       padding: 0,
       selectors: {
         '&:hover .ms-Button': {
-          backgroundColor: palette.neutralLighter
+          backgroundColor: semanticColors.buttonBackground
         },
         '&:hover .ms-Button-icon': {
-          color: palette.neutralPrimary
+          color: semanticColors.buttonText
```

Comment:
Shouldn't this slot mention the word `hover`?

Reply:
@betrue-final-final the toolkit shows this to be #000000 can you confirm this should be #201F1E (buttonTextHovered)

Reply:
you shouldn't be looking at the hex colors that are assigned to the semantic slots, that's the point of semantic slots. they tell you how they should be used.
you should find the correct button paradigm to use here, like the action buttons perhaps, or links, depending on what the designer says, and then use the semantic slots from that category.